### ISSUE TITLE
Solving coupling capacitance

### ISSUE THREAD

**Issue by endwkd** [ydc123/CNNCap#2]
Hello! We changed the default of parser.add_argument('--goal', type=str, default='env', choices=['total', 'env', 'all']) to 'env', At this time, y_pred should predict the coupling capacitance value, and ys corresponds to the true value of the coupling capacitance, but our debug code finds that the value does not correspond,this problem is reflected in for i, (xs, ys, masks, ys_total) in enumerate(tqdm(train_loader)):  The value of xs and ys taken out is different from the corresponding value in train_loader. How can I get the value of coupling capacitance?Thank you!!
![Uploading code.png…]()

**Comment by ydc123** [comment]
Please describe your question more precisely.

**Comment by endwkd** [comment]
Debug this line of code: for i, (xs, ys, masks, ys_total) in enumerate(tqdm(train_loader)):  We find that ys is not a value taken from train_loader. We would like to know whether this is deliberately set by you,or what's the point of that? Thank you!!

**Comment by ydc123** [comment]
Obviously, ys is taken from the train_loader (see https://github.com/ydc123/CNNCap/blob/main/dataset.py#L42)

**Comment by endwkd** [comment]
Sorry, it is our mistake, thank you for your reply, have a good day!!
